src/OpenToolkit.Mathematics/Geometry/Box3d.cs

```diff
@@ -14,161 +14,310 @@
 namespace OpenToolkit.Mathematics
 {
     /// <summary>
-    /// Defines a 3d box.
+    /// Defines a 2d box (rectangle).
```

Comment:
perhaps additional information, that it is axis aligned? Same for the other boxes.